apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -4949,8 +4947,7 @@ expression: "&schema"
                       "properties": {
                         "endpoint": {
                           "description": "The endpoint to send reports to",
-                          "type": "string",
-                          "format": "uri"
```

Comment:
I may follow up to reintroduce `format` into the schema.

Reply:
Actually, this is a mistake. We accept default, which is not a URI, so this must just be a string.